### ISSUE TITLE
Circle CI VSCode extension keeps writing to .vscode/settings.json

### ISSUE THREAD

**Issue by dalldrit** [circleci/circleci-docs#8751]
# Description
Starting recently the CircleCI extension has been writing to my .vscode/settings.json file and I cannot find a way to disable it, so I've just had to uninstall the extension. 

# Location
I started running into issues with merging when pulling from remote or when switching branches
I can see settings being added to my settings file and if I undo them, it instantly adds them back

Examples of the settings it's writing
```json
{
  "circleci.filters.branchFilter": "currentBranch",
  "circleci.persistedProjectSelection": []
}
```
Not sure if this is intentional behaviour thats been introduced but it's rather annoying

# Steps to Reproduce
Install and authenticate the circleci vscode extension and open up VSCode with a git repo that is connected to a circle ci project

**Comment by JulesFaucherre** [comment]
Hey @dalldrit, which version of the extension do you have. We released a new version this morning, `2.8.1` can you try it?

**Comment by dalldrit** [comment]
Hi @JulesFaucherre, just reinstalled and looks like the problem has been removed. Thank you!
